permissions/plugin-global-pre-script.yml

```
@@ -0,0 +1,7 @@
+---
+name: "global-pre-script"
+github: "jenkinsci/global-pre-script-plugin"
+paths:
+- "org/jenkins-ci/plugins/global-pre-script"
```

This should be the groupId from your pom (outside the <parent> section) / artifactId
```suggestion
- "io/jenkins/plugins/global-pre-script"
```